pkg/util/filesystem/util_unix.go

```diff
@@ -35,3 +36,8 @@ func IsUnixDomainSocket(filePath string) (bool, error) {
 	}
 	return true, nil
 }
+
+// IsAbs is same as filepath.IsAbs on Unix.
+func IsAbs(path string) bool {
```

Comment:
nit: I think we can simply set ``IsAbs := filepath.IsAbs``, which means that we'd avoid a call on the call stack, similar to how I've done it here: https://github.com/kubernetes/kubernetes/pull/112566/files#diff-2a196acccdccf1b0104a41a86a41a986adff8c975a9d608a9d2dbee96c38c6e9R41

Reply:
I asked for the function to avoid a mutable package variable in an earlier review